### PR TITLE
update minor versions

### PR DESCRIPTION
**Pull request by Ryanmtate**
Updates for minor versions

### CODE REVIEW

File: nj-core/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "nj-core"
-version = "2.1.1"
+version = "2.1.2"
```
Comment (sehz) [inline nj-core/Cargo.toml:3]: don't think there is update in nj-core

File: nj-derive/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "nj-derive"
-version = "2.0.1"
+version = "2.0.2"
```
Comment (sehz) [inline nj-derive/Cargo.toml:3]: no changes here

File: nj-sys/Cargo.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "nj-sys"
-version = "1.0.0"
+version = "1.0.1"
```
Comment (sehz) [inline nj-sys/Cargo.toml:3]: no changes in nj-sys

File: nj-core/Cargo.toml (hunk 1 above)
Comment (Ryanmtate) [reply nj-core/Cargo.toml]: daea9a6 reverts

File: nj-derive/Cargo.toml (hunk 2 above)
Comment (Ryanmtate) [reply nj-derive/Cargo.toml]: daea9a6 reverts

File: nj-sys/Cargo.toml (hunk 3 above)
Comment (Ryanmtate) [reply nj-sys/Cargo.toml]: daea9a6 reverts

File: Cargo.toml [hunk 4]
```diff
@@ -16,6 +16,6 @@ build = ["nj-build"]
 
 [dependencies]
 nj-sys = { path = "nj-sys", version = "1.0.0", optional = true }
-nj-core = { path = "nj-core", version = "2.1.0", optional = true  }
-nj-build = { path = "nj-build", version = "0.2.0", optional = true }
-nj-derive = { path = "nj-derive", version = "2.0.0", optional = true}
+nj-core = { path = "nj-core", version = "2.1.1", optional = true  }
```
Comment (Ryanmtate) [inline Cargo.toml:19]: I bumped these version in the main Cargo.toml file to be equal to their crate versions

**Pull request merged**